docs/source/how-to-guides/load-your-data/open-save-sqlite-file.rst

```diff
@@ -1,12 +1,23 @@
-===================================
-How-To: Open and Save a SQLite File
-===================================
+====================================
+:index:`Open and Save a SQLite File`
+====================================
 
 Open a SQLite File
 ==================
 
+Once you've imported your JSON scan and saved it as SQLite, in most cases you'll continue your
+review and analysis work solely in the SQLite file.
```

Comment:
I would get rid of "in most cases". Whether the user wants it or not, all the reviewing and analysis work is done via the sqlite file (by design).

Reply:
fixed